app/models/order.rb

```diff
@@ -6,6 +6,7 @@ class Order < ApplicationRecord
     "Purchase order" => 2
   }
   
+  belongs_to :user
```

Comment:
should there be any extra configuration here like `dependent` or something else

Reply:
No, as there is already a dependent option in user.rb for ```has_many :orders``` . Having dependent on both sides  an lead to orphaned records in the database.